administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -22,7 +22,7 @@ function iframeRef( frameRef ) {
 
 	function triggerSave() {
 		var inside = iframeRef( document.getElementById('target-association') );
-		inside.getElementById('applyBtn').click();
+		window.frames[1].Joomla.submitbutton('article.apply');
```

Comment:
so, use the normal layout and hide the toolbar? because the normal layout brings the page's header